src/components/Footer/Footer.scss

```
@@ -0,0 +1,403 @@
+// @media (max-width:1200px) {
+.footerContainer {
+  .iconWrapper {
+    .iconList {
+      display: flex;
+      justify-content: space-around;
```

```suggestion
.footerContainer {
    .iconList {
      display: flex;
      justify-content: space-around;
```
스타일이 없는 셀렉터까지 적어줄 필요 없습니다